src/components/Shared/AddressForm.js

```diff
@@ -5,7 +5,7 @@ import { RequestsContext } from 'src/providers/RequestsContext'
 import toast from 'react-hot-toast'
 import { ControlContext } from 'src/providers/ControlContext'
 
-const AddressForm = ({ recordId, address, setAddress = () => {}, editMode, onSubmit }) => {
+const AddressForm = ({ recordId, address, setAddress = () => {}, editMode, onSubmit, isCleared = true }) => {
```

Comment:
Call it clearable. Let whatever is passed to plant window get passed to FormShell.